configuration/concepts/commons.csv

```diff
@@ -1,22 +1,57 @@
-334dbd41-8a92-11e4-977f-0800271c1b75,,CIEL:160870,Q6H,Q6H,Q6H,Q6H,,,Frequency,N/A,,,,,,,,,,,
-334ea16c-8a92-11e4-977f-0800271c1b75,,CIEL:160866,Q8H,Q8H,Q8H,Q8H,,,Frequency,N/A,,,,,,,,,,,
-334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,Q12H,Q12H,Q12H,,,Frequency,N/A,,,,,,,,,,,
+334dbd41-8a92-11e4-977f-0800271c1b75,,CIEL:160870,Q6H,QID,Q6H,QID,,,Frequency,N/A,,,,,,,,,,,
+334ea16c-8a92-11e4-977f-0800271c1b75,,CIEL:160866,Q8H,TDS,Q8H,TDS,,,Frequency,N/A,,,,,,,,,,,
+334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,BD,Q12H,BD,,,Frequency,N/A,,,,,,,,,,,
 335071ea-8a92-11e4-977f-0800271c1b75,,,Q48H,Q48H,Q48H,Q48H,,,Frequency,N/A,,,,,,,,,,,
+90233a68-c4c1-48fb-a210-2cb0d5544e00,,,Five times/Day,Five times/Day,Five times/Day,Five times/Day,,,Frequency,N/A,,,,,,,,,,,
+863264e3-1b95-441a-997d-7fb0a240613c,,,Q1H,Q1H,Q1H,Q1H,,,Frequency,N/A,,,,,,,,,,,
+d4dc0576-7a0d-43da-a123-19b26119cc6a,,,Q2H,Q2H,Q2H,Q2H,,,Frequency,N/A,,,,,,,,,,,
+4adb2e4e-40c8-4889-81ac-99b940897f54,,,Q3H,Q3H,Q3H,Q3H,,,Frequency,N/A,,,,,,,,,,,
+2c4b7e6b-1bdd-4fe9-abf0-ced777b6caf9,,,Q5H,Q5H,Q5H,Q5H,,,Frequency,N/A,,,,,,,,,,,
+8116d2c6-a183-4132-900d-02cc54a1821a,,,Monthly,Monthly,Monthly,Monthly,,,Frequency,N/A,,,,,,,,,,,
+b22ec437-b389-4875-86f9-9662732e384d,,,Twice/Week,Twice/Week,Twice/Week,Twice/Week,,,Frequency,N/A,,,,,,,,,,,
+3a427f19-979b-43f4-8e91-0fbdcf4a3773,,,Thrice/Week,Thrice/Week,Thrice/Week,Thrice/Week,,,Frequency,N/A,,,,,,,,,,,
+9261ebac-541a-4a96-b245-07b9adcccb99,,,Once Every 2 Weeks,Once Every 2 Weeks,Once Every 2 Weeks,Once Every 2 Weeks,,,Frequency,N/A,,,,,,,,,,,
+023570b0-872f-414d-88f1-6050b39e79f3,,,Once Every 3 Weeks,Once Every 3 Weeks,Once Every 3 Weeks,Once Every 3 Weeks,,,Frequency,N/A,,,,,,,,,,,
 33516e22-8a92-11e4-977f-0800271c1b75,,CIEL:1099,Weekly,Hebdomadaire,Weekly,Hebdomadaire,,,Frequency,N/A,,,,,,,,,,,
-9d8e6918-3f10-11e4-adec-0800271c1b75,,CIEL:161235,Now,Maintenant,STAT,STAT,,,Frequency,N/A,,,,,,,,,,,
 2a148983-c143-4649-b4be-f8845c934ec0,,,Nightly,Chaque nuit,Nightly,Chaque nuit,,,Frequency,N/A,,,,,,,,,,,
+f36097e7-7d66-4196-8d95-6b66f31435cc,,,See comments,Commentaires,See comments,Commentaires,,,Frequency,N/A,,,,,,,,,,,
+9d5b708e-3f10-11e4-adec-0800271c1b75,,CIEL:162384,Dosing Units,Unités de dosage,dosing units,Unités de dosage,,,ConvSet,N/A,,,Capsule(s); Tablet(s); ml; mg; IU; Drop(s); Tablespoon; Teaspoon; Unit(s); Vial(s); Ampule(s); Puff(s); Application; Packet; Dose; See comments,,,,,,,,
+0de96a2d-5c2c-4110-9770-914369de1713,,CIEL:162390,Opthalmic,Ophtalmique,Opthalmic,Ophtalmique,,,Misc,N/A,,,,,,,,,,,
+99cbeb1b-621c-4645-a600-d702364d9885,,CIEL:162387,Otic,Otique,Otic,Otique,,,Misc,N/A,,,,,,,,,,,
+9d6c9153-3f10-11e4-adec-0800271c1b75,,CIEL:162392,Intravaginal,Intravaginale,Intravaginal,Intravaginale,,,Misc,N/A,,,,,,,,,,,
+c5dee13f-524c-4dcc-9ad5-478a77596096,,CIEL:160240,Oral,Oral,Oral,Oral,,,Misc,N/A,,,,,,,,,,,
+1c0e27f8-63af-4bba-9629-fae799948a7c,,,Topical,Topical,Topical,Topical,,,Misc,N/A,,,,,,,,,,,
```

Comment:
This one I am not sure, I'll ask around and let you know.

Reply:
ok, in fr that'd be "topique"

Reply:
Done